components/navigation/navigation.js

```diff
@@ -0,0 +1,66 @@
+import React from 'react'
+import Link from "next/link";
+import styles from './navigation.scss'
+
+
+class Navigation extends React.Component {
+
+
+    constructor(props) {
+        super(props);
+    }
+
+    render() {
+        return (
+            <div>
+                <style dangerouslySetInnerHTML={{__html: styles}}/>
+                <div className="nav buttons">
+                    <a href="https://medium.com/nteract" target="_blank" className="nav-item">
+                        Desktop
+                    </a>
+                    <a href="https://medium.com/nteract" target="_blank" className="nav-item">
+                        Atom
+                    </a>
+                    <Link as="/kernels" >
+                        <a href="/kernels" prefetch className="nav-item">Kernels</a>
+                    </Link>
+                    <div className="icon-buttons buttons">
+                        <a href="https://slackin-nteract.now.sh/" target="_blank" className="button button-secondary button-icon-only" title="Join us on Slack!">
+                            <div className="button-wrapper">
+                                <div className="button-icon">
```

Comment:
To make this easier for reading, it would be cool to make each of the svgs into individual named functions:

```
const GitHubIcon = () => <svg> ...
```